languages.ts

```diff
@@ -552,4 +552,38 @@ export const languageSpecs: LanguageSpec[] = [
         },
         stylized: 'R',
     },
+    {
+        handlerArgs: {
+            languageID: 'pascal',
+            fileExts: ['p', 'pas', 'pp'],
+            commentStyle: {
+                // Traditional: (* this is a comment *)
+                // Customary:   { this is also a comment }
+                block: {
+                    startRegex: /(\{|\(\*)\s?/,
+                    endRegex: /(\}|\*\))/,
+                },
+
+                // TODO: Some Pascal implementations support //-comments too.
+                // Is that common enough to support here?
+            },
+        },
+        stylized: 'Pascal',
+    },
+    {
+        handlerArgs: {
+            languageID: 'verilog',
+            fileExts: ['sv', 'svh', 'svi', 'v'],
+            commentStyle: cStyle,
+        },
+        stylized: 'Verilog',
+    },
+    {
+        handlerArgs: {
+            languageID: 'vhdl',
```

Comment:
Also needs an entry in https://github.com/sourcegraph/sourcegraph/blob/3a1dd8ccd49175ef2ed47dabe85d2e0543ce6c47/shared/src/languages.ts#L40

Reply:
Yes, thanks, that's part of https://github.com/sourcegraph/sourcegraph/pull/8079.